tests/actions/test_validation_operators.py

```diff
@@ -105,16 +106,16 @@ def warning_failure_validation_operator_data_context(
     data_context.update_expectation_suite(expectation_suite=warning_expectations)
 
     failure_expectations.expectation_suite_name = "f2.failure"
-    data_context.add_expectation_suite(expectation_suite=failure_expectations)
+    data_context.suites.add(ExpectationSuite(name=failure_expectations))
```

Comment:
Looks like this and the next 3 are passing an ExpectationSuite as the name, unless I'm missing something.

Reply:
i think this is actually correct, the signature is `SuiteFactory.add(suite: ExpectationSuite)` ([source](https://github.com/great-expectations/great_expectations/blob/4ce453be08c43c36b8b7c6a77fe1eaada8821ee1/great_expectations/core/suite_factory.py#L24))

Reply:
ah, thanks, you're right!